karma.conf.js

```diff
@@ -48,7 +48,7 @@ module.exports = function(config) {
       'app/bower_components/angularfire/dist/angularfire.js',
       'app/bower_components/restangular/dist/restangular.js',
       'app/bower_components/moment/moment.js',
-      'app/bower_components/moment-timezone/builds/moment-timezone-with-data.js',
+      'app/bower_components/moment-timezone/builds/moment-timezone-with-data-2012-2022.js',
```

Comment:
The new bower distribution of `moment-timezone` comes with that specific of a filename distribution?